.github/workflows/continuous-integration.yml

```diff
@@ -118,17 +118,17 @@ jobs:
             - name: Link
               run: npm link
 
-            - name: combineJavascript main
+            - name: cplace-asc main
               working-directory: ./main
               timeout-minutes: 20
-              run: ./gradlew combineJavascript
+              run: cplace-asc --verbose
```

Comment:
why is gradle no longer involved ?

Reply:
I'd guess that's due to: we try to separate the gradle and npm stuff as good as possible. so IIRC the gradle plugin just kicks of the asc compilation to make sure the assets are part of the software.zip but does not combine the JavaScripts any longer or has other 'duties' considering frontend compilation.

Reply:
Correct. Vladimir added this change to cplace-asc few releases back